cypress/integration/frontpageLayoutSpec.js

```diff
@@ -25,7 +25,10 @@ Object.keys(services).forEach(index => {
         it('should have a visible banner', () => {
           cy.get('header')
             .should('have.lengthOf', 1)
-            .find('a')
+            .find('div[class^="Banner"]')
```

Comment:
I had to add this because moving the ConsentBanner into this container broke the test because there are now multiple links in the container.